spharpy/samplings/coordinates.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
-from spharpy.samplings.helpers import sph2cart
+from pyfar.classes.coordinates import sph2cart, cyl2cart
+import pyfar as pf
+from spharpy.samplings.helpers import sph2cart as sp_sph2cart
 from scipy.spatial import cKDTree
-import pyfar
 
 
 class Coordinates(object):
```

Comment:
I think we should add a deprecation warning to this, or did you plan on doing that on a different branch?